### PR TITLE
Coverage now built into tests

### PR DESCRIPTION
**Pull request by DarkPurple141**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 .DS_Store
 node_modules
 /dist
+coverage
+.nyc_output
```
Comment (NunoDasNeves) [inline .gitignore]: we're outputting to new york city?

File: backend/package.json [hunk 2]
```diff
@@ -17,9 +18,11 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
+    "coveralls": "^3.0.2",
     "eslint": "^5.3.0",
     "mocha": "^5.2.0",
-    "nodemon": "^1.18.3"
+    "nodemon": "^1.18.3",
```
Comment (NunoDasNeves) [inline backend/package.json]: This nodemon thing seems neat

File: backend/package.json [hunk 3]
```diff
@@ -17,9 +18,11 @@
...
+    "nyc": "^13.0.1"
```
Comment (NunoDasNeves) [inline backend/package.json]: nyc is the thing that actually checks code coverage yeah?

File: backend/package.json [hunk 4]
```diff
@@ -17,9 +18,11 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
+    "coveralls": "^3.0.2",
```
Comment (NunoDasNeves) [inline backend/package.json]: coveralls adds the coverage button thing to README, and does...what else?

**Comment by DarkPurple141** [comment]
@NunoDasNeves Please approve and merge k ty.

**Pull request merged**